extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -422,10 +422,10 @@ function tests(name) {
     it('should add container info when ad has a container', () => {
       return getAdInAdContainer().then(obj => {
         const ampAd = obj.ampAd;
+        const impl = ampAd.implementation_;
         expect(ampAd.getAttribute('amp-container-element')).to.be.null;
-        ampAd.implementation_.onLayoutMeasure();
-        expect(ampAd.getAttribute('amp-container-element'))
-            .to.equal('AMP-STICKY-AD');
+        impl.onLayoutMeasure();
+        expect(impl.container_.tagName).to.equal('AMP-STICKY-AD');
```

Comment:
I don't know what to put as the object prototype to stub an import global function like `#getIframe`. Any idea on this?